### DISCUSSION TITLE
Preview URL does not work after upgrading to 1.8.1

### DISCUSSION THREAD

**Discussion by ookris** [YOURLS/YOURLS#2922] [Plugins]
Hi. 
Can anyone take a look and see why the Preview URL (https://github.com/YOURLS/YOURLS/wiki/Plugin-=-Preview-URL) plugin is not working properly with the new version of the script?
After updating, the preview page shows a 404 error

**Comment by ozh** [comment]
You have another plugin to deal with generating a 404 or something when a short url is not found ?

**Comment by ookris** [comment]
Yes, I have 404-Redirect but I don't remember if this plugin was turned on after the update.

---edit---

I did a clean install and turned on the plugins I was using one by one. It turned out that the 404-Redirect plugin was causing the problem.

**Reply by ozh** [reply]
Yes, there is a hook conflict. I'll see if I can tweak the Preview plugin to make both co exist when I have some time

**Reply by arladmin** [reply]
Hi @ozh 

I'm facing the same issue.
Have both the plugins installed: 404 & preview.

But the Preview Plugin hasn't worked even once. On hovering over the shorturl in a webpage, nothing happens. And on clicking, i get a 404 error.
